websites/V/Vrc.tl/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "937757295453044806",
+		"name": "atsukoro"
+	},
+	"service": "Vrc.tl",
```

Comment:
Ideally, the folder should be properly capitalized. (`vrc.tl`), unless the service also describes itself using the current capitalization (`Vrc.tl`)
Here are some resources that could help with adjusting the capitalization.
* https://stackoverflow.com/a/16071375/5844241
* https://stackoverflow.com/a/3011723/5844241